pkg/reconciler/sinkbinding/controller.go

```diff
@@ -114,6 +135,20 @@ func NewController(
 		},
 	}
 
+	// Reconcile SinkBinding when the OIDC service account changes
+	serviceaccountInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: controller.FilterController(&v1.SinkBinding{}),
+		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
+	})
+
+	// reconcile sinkindings on changes on the features configmap
+	configmapInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: controller.FilterWithNameAndNamespace(system.Namespace(), feature.FlagsConfigName),
+		Handler: controller.HandleAll(func(i interface{}) {
+			impl.GlobalResync(sbInformer.Informer())
+		}),
+	})
```

Comment:
Instead of using an informer, we can configure a callback function when calling `NewStore`, that saves quite a bit of memory since that watcher is watching only configmaps in the system namespace as opposed to watch every configmap in the cluster /cc @creydr

Reply:
see example in `pkg/reconciler/broker/controller.go`

Reply:
like `featureStore.WatchConfigs(cmw)` , right ?

Reply:
yes. `featureStore.WatchConfigs(cmw)` takes additional callback functions which get called on configmap updates. E.g.: 
```
var globalResync func(obj interface{})
featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"), func(name string, value interface{}) {
	if globalResync != nil {
		globalResync(nil)
	}
})
```
with `globalResync` being something like:
```
globalResync = func(obj interface{}) {
	impl.GlobalResync(sbInformer.Informer())
}
```

Reply:
thanks @creydr